mne/preprocessing/_fine_cal.py

```diff
@@ -43,6 +44,9 @@ def compute_fine_calibration(
     origin=(0.0, 0.0, 0.0),
     cross_talk=None,
     calibration=None,
+    *,
+    angle_limit=5.0,
+    err_limit=5.0,
```

Comment:
AFAICT `err_limit` isn't validated anywhere, there should at least be some check that it is non-negative right?